src/notifiers/TransferRewardNotifier.sol

```diff
@@ -55,6 +55,6 @@ contract TransferRewardNotifier is RewardTokenNotifierBase {
   /// using transfer. This contract must have a sufficient balance of reward tokens for the
   /// transfer to succeed.
   function _sendTokensToReceiver() internal virtual override {
-    TOKEN.transfer(address(RECEIVER), rewardAmount);
```

Comment:
(arbitrary comment)

- Was there a specific reason we opted out of  adding the `safeApprove` call in the `TransferRewardNotifier`.

Reply:
Good catch! I'm just an idiot. Fixed.